### PR TITLE
Starting with documentation

### PR DESCRIPTION
**Pull request by mikkjant**
Start of the documentation.

### CODE REVIEW

**Comment by ernoaapa** [comment]
Build fails because there's `install-istio.md` reference but there's no file with that name.

File: www/content/replicaset.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ReplicaSet
+
+Usually one does not create pods directly but create other resources like ReplicaSet's which creates and manages pods.
+(This because if node fails then those pods are lost.)
+
+Keeping pods healthy
+
+```shell
+
+workshop $ kubectl
```
Comment (ernoaapa) [inline www/content/replicaset.md]: If the "ReplicaSet" content is not done, maybe remove this file from this PR and create new PR when it's completed.

**Review by ernoaapa — CHANGES_REQUESTED** [review]
Maybe either remove the www/content/replicaset.md from this PR or finish it up.

**Comment by mikkjant** [comment]
Modification done.

**Review by ernoaapa — APPROVED** [review]

**Pull request merged**